cdhweb/static_src/global/components/tiles.scss

```diff
@@ -48,6 +57,7 @@
 .tile__text {
   grid-row: 2;
   grid-column: 1;
+  background-color: var(--color-white);
```

Comment:
Because the component bg can now be grey. But the tile should always have a white bg.